src/containers/ArticlePage/ArticlePage.jsx

```diff
@@ -120,6 +128,79 @@ class ArticlePage extends Component {
 
     const article = transformArticle(resource.article, locale);
 
+    const activeFilterId = getFiltersFromUrl(location);
+    const filter = subject.filters.filter(filter =>
+      activeFilterId.split(',').includes(filter.id),
+    );
+    const [mainTopic, subTopic] = topicPath;
+
+    const handleNav = (e, item) => {
+      e.preventDefault();
+      const { id } = item;
+      if (id !== article.id) {
+        const breadCrumbIds = breadCrumbs.map(crumb => crumb.id);
+        const breadCrumbsWithoutFilter = breadCrumbIds.filter(
+          c => c !== filter?.id,
+        );
+        history.push(
+          toTopic(
+            breadCrumbsWithoutFilter[0],
+            activeFilterId,
+            ...breadCrumbsWithoutFilter.slice(
+              1,
+              breadCrumbsWithoutFilter.indexOf(id) + 1,
+            ),
+          ),
+        );
+      }
+    };
+
+    const breadCrumbs = [
+      {
+        id: subject.id,
+        label: subject.name,
+        typename: 'Subjecttype',
+        url: '#',
+      },
+      ...(filter
+        ? [
+            {
+              id: filter.id,
+              label: filter?.map(f => f.name)?.reduce((a, b) => a + ', ' + b),
```

Comment:
Denne linja kaster en exeption ved visning av artikkel i PR-installasjonen.

Reply:
TypeError: Reduce of empty array with no initial value
    at Array.reduce (<anonymous>)
    at n.value (ArticlePage.jsx:169)